doc/filtering.rst

```diff
@@ -95,7 +95,7 @@ This section contains functions to filter values during iteration.
    :returns: {gen1, param1, state1}, {gen2, param2, state2}
 
    The function returns two iterators where elements do and do not satisfy the
-   predicate. Equivalent to:
+   predicate. See note about multireturn in :doc:`getting_started`. Equivalent to:
```

Comment:
I hope there is a way to set an anhor and link it here. Looks quite unconvenient to find needed recipe over the doc, when you don't know what exactly was meant.

(I know, it was deliberate decision, but) I would place the right here (somewhere below examples) and link from `span`. I don't mind a general note and linking it from both functions, but I see no appropriate place for such note.

Reply:
> I hope there is a way to set an anhor and link it here. Looks quite unconvenient to find needed recipe over the doc, when you don't know what exactly was meant.

Added anchor.